service/src/intTest/java/uk/nhs/adaptors/gp2gp/ehr/IllogicalMessageComponentTest.java

```diff
@@ -264,20 +259,11 @@ private static String asString(Resource resource) {
 
     @SneakyThrows
     private void mockIncomingMessage(String ebxml, String payload, String interactionId, String conversationId) {
```

Comment:
"payload", "interactionId", "conversationId" input params in "mockIncomingMessage" method are not used and are ignored.